document/static/js/modules/editor/navigator/index.js

```diff
@@ -1,10 +1,13 @@
 import {escapeText ,findTarget} from "../../common"
+import {navigatorFilterModel} from "./model"
 export class ModNavigator {
     constructor(editor) {
         editor.mod.navigator = this
         this.editor = editor
         this.navigatorEl = document.querySelector('#navigator')
         this.listeners = {}
+        this.navigatorFilters = navigatorFilterModel().content
```

Comment:
Can we mount this on the editor and then refer to it there? That way it's possible to add/remove entries to the menu from plugins.